test/cache_assignment_logger_test.py

```diff
@@ -0,0 +1,112 @@
+from unittest.mock import Mock
+
+from cachetools import LRUCache
+
+from eppo_client.assignment_logger import AssignmentCacheLogger
+from eppo_client.client import _utcnow
+from eppo_client.version import __version__
+
+
+def test_non_caching():
+    inner = Mock()
+    logger = AssignmentCacheLogger(inner)
+
+    logger.log_assignment(mk_assignment_event())
+    logger.log_assignment(mk_assignment_event())
+    logger.log_bandit_action(mk_bandit_event())
+    logger.log_bandit_action(mk_bandit_event())
+
+    assert inner.log_assignment.call_count == 2
+    assert inner.log_bandit_action.call_count == 2
+
+
+def test_assignment_cache():
+    inner = Mock()
+    logger = AssignmentCacheLogger(inner, assignment_cache=LRUCache(100))
+
+    logger.log_assignment(mk_assignment_event())
+    logger.log_assignment(mk_assignment_event())
+
+    assert inner.log_assignment.call_count == 1
+
+
+def test_bandit_cache():
+    inner = Mock()
+    logger = AssignmentCacheLogger(inner, bandit_cache=LRUCache(100))
+
+    logger.log_bandit_action(mk_bandit_event())
+    logger.log_bandit_action(mk_bandit_event())
+
+    assert inner.log_bandit_action.call_count == 1
+
+
+def test_bandit_flip_flop():
+    inner = Mock()
+    logger = AssignmentCacheLogger(inner, bandit_cache=LRUCache(100))
+
+    logger.log_bandit_action(mk_bandit_event(action="action1"))
+    logger.log_bandit_action(mk_bandit_event(action="action1"))
+    assert inner.log_bandit_action.call_count == 1
+
+    logger.log_bandit_action(mk_bandit_event(action="action2"))
+    assert inner.log_bandit_action.call_count == 2
+
+    logger.log_bandit_action(mk_bandit_event(action="action1"))
+    assert inner.log_bandit_action.call_count == 3
+
+
+def mk_assignment_event(
```

Comment:
super minor, but consider spelling out `mock` (instead of `mk`), I don't think the few characters saved is worth the extra cognitive step for developers reading the code to get what is going on

Reply:
`mk` here actually stands for "make" (as in `mkdir`). Thought it's quite widely understood but it's probably just me coming from C background and writing too much makefiles (`.mk`) 😅 

I'll spell it out